testutils/testwallet.go

```diff
@@ -327,9 +333,16 @@ func (w *testWallet) createTx(outputs []*types.TxOutput, feePerByte types.Amount
 		totalOutAmt[o.Amount.Id] += o.Amount.Value
 		tx.AddTxOut(o)
 	}
+
+	// Set the Locktime, if given.
+	if lockTime != nil {
+		tx.LockTime = uint32(*lockTime)
+	}
+
 	enoughFund := false
 	// select inputs from utxo set of the wallet && add them into tx
 	for txOutPoint, utxo := range w.utxos {
+		fmt.Println(txOutPoint.Hash.String(), txOutPoint.OutIndex, utxo.value.Id.Name(), utxo.value.Value)
```

Comment:
avoid of using `fmt.Println`,  remove it or replace by `log`

Reply:
127626c